src/app/reusableComponents/NewsList/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
I cannot accept the existing folder and file structure, please check the official docs for further info

Reply:
Done.
Application is divided into components, containers, constants, and utilities.